presto-docs/src/main/sphinx/presto_cpp/features.rst

```diff
@@ -256,3 +256,10 @@ Users can enable collection of worker level metrics by setting the property:
 
   When enabled and Presto C++ workers interact with the S3 filesystem, additional runtime metrics are collected.
   For a detailed list of these metrics, see `S3 FileSystem <https://facebookincubator.github.io/velox/monitoring/metrics.html#s3-filesystem>`_.
+
+``runtime-histogram-metrics-collection-enabled``
```

Comment:
The issue is specific to prometheus. Meta's implementation does not have this issue.
Let's name this `prometheus-histogram-metrics-collection-enabled`. Move the documentation close to prometheus.